package.json

```diff
@@ -54,6 +54,7 @@
   },
   "dependencies": {
     "@metamask/eth-sig-util": "^5.0.0",
+    "@nomicfoundation/hardhat-toolbox": "^2.0.0",
```

Comment:
remove

Reply:
Im not sure what do you mean with by remove, well I do but not pretty sure.